app/notifiers/activity_wrapper.rb

```diff
@@ -0,0 +1,27 @@
+class ActivityWrapper < SimpleDelegator
```

Comment:
This should not live in notifiers because notifiers are really just for notifiers. this is not a notifier.